icon/iiss/icstate/term.go

```diff
@@ -397,9 +393,9 @@ func newTermWithTag(_ icobject.Tag) *Term {
 	return &Term{}
 }
 
-func newTerm() *Term {
+func newTerm(termPeriod int64) *Term {
 	return &Term{
-		period:         termPeriod,
+		period:         int(termPeriod),
```

Comment:
Please change the type of period from int to int64.

Reply:
I changed the type of termPeriod to int64 and added nil check
Thank you